spark-connector-examples/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: "java"
```

Comment:
None of these files contains the Pravega license header. Please add them, according to the format of this repo: https://github.com/pravega/pravega